python/ray/function_manager.py

```diff
@@ -539,6 +550,8 @@ def export_actor_class(self, Class, actor_method_names,
             # because of https://github.com/ray-project/ray/issues/1146.
 
     def load_actor(self, driver_id, function_descriptor):
+        print("Load Actor: function_descriptor: %s" % function_descriptor)
+        print("Load Actor: function_id: %s" % function_descriptor.function_id)
```

Comment:
all of the print statements should be removed before merging (or changed to `logger.debug`)